app/src/main/java/com/sunasterisk/travelapp/ui/adapter/RestaurantTabAdapter.kt

```diff
@@ -0,0 +1,56 @@
+package com.sunasterisk.travelapp.ui.adapter
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.sunasterisk.travelapp.R
+import com.sunasterisk.travelapp.data.models.Location
+import com.sunasterisk.travelapp.ui.base.BaseRecyclerAdapter
+import com.sunasterisk.travelapp.ui.base.BaseViewHolder
+import com.sunasterisk.travelapp.ui.adapter.RestaurantTabAdapter.LocationViewHolder
+import com.sunasterisk.travelapp.utils.setImageUrl
+import kotlinx.android.synthetic.main.item_destination.view.*
+
+class RestaurantTabAdapter(private var onItemClick: (Location) -> Unit) :
+    BaseRecyclerAdapter<Location, LocationViewHolder>() {
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): LocationViewHolder {
+        val itemView = LayoutInflater.from(parent.context)
+            .inflate(R.layout.item_destination, parent, false)
+        return LocationViewHolder(itemView, onItemClick)
+    }
+
+    class LocationViewHolder(
+        itemView: View,
+        onItemClick: (Location) -> Unit
+    ) : BaseViewHolder<Location>(itemView, onItemClick) {
+        override fun onBindData(itemData: Location) {
+            super.onBindData(itemData)
+            with(itemView) {
+                textViewDestinationNameItem.text = itemData.name
+                when (itemData.type) {
+                    Location.TYPE_GEO -> {
+                        textViewPropertiesItem.text =
+                            resources.getQuantityString(
+                                R.plurals.title_properties,
+                                resources.getInteger(R.integer.integer_1), itemData.propertiesNumber
+                            )
+                    }
+                    Location.TYPE_RESTAURANT -> {
+                        textViewPropertiesItem.text =
+                            "${resources.getString(R.string.title_restaurant_address)} ${itemData.location}"
```

Comment:
https://stackoverflow.com/questions/12627457/format-statement-in-a-string-resource-file
```suggestion
                        textViewPropertiesItem.text =
                            resources.getString(R.string.title_restaurant_address, itemData.location)
```

Reply:
![image](https://user-images.githubusercontent.com/53286739/93428490-c1c1e680-f8e9-11ea-98f6-0bdec0613517.png)
Thụt lề sâu quá, tạo các biến phụ